osc-server/src/main/java/org/osc/core/broker/model/plugin/manager/SecurityGroupInterfaceElementImpl.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.model.plugin.manager;
+
+import java.util.Set;
+
+import org.osc.sdk.manager.element.SecurityGroupInterfaceElement;
+
+public class SecurityGroupInterfaceElementImpl implements SecurityGroupInterfaceElement {
+
+	String securityGroupInterfaceId;
+	String name;
+	String managerSecurityGroupId;
+	Set<String> managerPolicyIds;
+	String tag;
+
+	public SecurityGroupInterfaceElementImpl(String securityGroupInterfaceId, String name, String managerSecurityGroupId,
+			Set<String> managerPolicyIds, String tag) {
+		super();
+		this.securityGroupInterfaceId = securityGroupInterfaceId;
+		this.name = name;
+		this.managerSecurityGroupId = managerSecurityGroupId;
+		this.managerPolicyIds = managerPolicyIds;
+		this.tag = tag;
+	}
+
+	@Override
+	public String getSecurityGroupInterfaceId() {
+		return this.securityGroupInterfaceId;
+	}
+
+	public void setId(String securityGroupInterfaceId) {
+		this.securityGroupInterfaceId = securityGroupInterfaceId;
+	}
+
+	@Override
+	public String getName() {
+		return this.name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	@Override
+	public String getManagerSecurityGroupId() {
+		return this.managerSecurityGroupId;
+	}
+
+	public void setManagerSecurityGroupId(String managerSecurityGroupId) {
+		this.managerSecurityGroupId = managerSecurityGroupId;
+	}
+
+	@Override
+	public Set<String> getManagerPolicyIds() {
+		return this.managerPolicyIds;
+	}
+
+	public void setManagerPolicyIds(Set<String> managerPolicyIds) {
+		this.managerPolicyIds = managerPolicyIds;
+	}
+
+	@Override
+	public String getTag() {
+		return this.tag;
+	}
+
+	public void setTag(String tag) {
```

Comment:
Again, if these are really needed check if they can be package private

Reply:
we cannot reduce the visibility of the inherited method